spark/AerospikeSparkPython.ipynb

```diff
@@ -0,0 +1,855 @@
+{
+ "cells": [
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "# Aerospike Connect for Spark Tutorial for Python\n",
+    "## Tested with Java 8, Spark 2.4.0, and Python 3.7"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 1,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "import os\n",
+    "os.environ[\"PYSPARK_SUBMIT_ARGS\"] = '--jars aerospike-spark-assembly-2.4.0.jar pyspark-shell'"
```

Comment:
Are we sure we want to directly put in the version of the jar here? It would mean we have to update with each release. Will that be needed regardless?

Reply:
Should I just say aerospike-spark-assembly-x.x.x.jar? and add a note that we tested with 2.4.0?